README.md

```diff
@@ -217,6 +217,24 @@ Stop the tunnel with: `make <env> stop-tunnel`
 ## Other useful commands
 Type `make` to get the list of all available commands.
 
+## Known caveats
+
+### Fly version
+
+Current mechanism of updating fly binary only checks if the date of your local
+binary is older than one on the server. If you have been using previous fly
+version recently, it won't get updated. In that case, delete the binary in /bin
```

Comment:
Does it not get updated by [`fly sync`](https://github.com/alphagov/paas-bootstrap/blob/4f42799af6c7a2c14188b701ea7bfbf852669baa/concourse/scripts/fly_sync_and_login.sh#L20-L21) (copied from and described in alphagov/paas-cf@3a2fd218ede057d2f3a32e12ef08c68018a889e1)?